webapp/packages/core-ui/src/SideBarPanel/SideBarPanel.tsx

```diff
@@ -42,9 +43,18 @@ const sideBarPanelRegistry: StyleRegistry = [
 
 export const SideBarPanel = observer<SideBarPanelProps>(function SideBarPanel({ container }) {
   const style = useS(styles);
+  const { onReorder, sortTabs, persistenceKey } = useTabOrderPersistence(container.areaLabel, () => container.getIdList());
```

Comment:
please don't use areaLable as persistence key, you can add property like `id` to the `SideBarPanel` and use it as persistence key